x-pack/plugins/profiling/public/utils/formatters/as_number.ts

```diff
@@ -11,18 +11,18 @@ export function asNumber(value: number): string {
   }
 
   value = Math.round(value * 100) / 100;
-  if (value < 0.01) {
+  if (value < 0.01 && value > -0.01) {
```

Comment:
Thanks @LucaWintergerst , my bad! Do you mind using `Math.abs(value) < ...`? I can also push it to your branch if you want to.